site/content/docs/5.0/examples/pricing/index.html

```diff
@@ -6,77 +6,145 @@
 include_js: false
 ---
 
-<header class="d-flex flex-column flex-md-row align-items-center p-3 px-md-4 mb-3 bg-body border-bottom shadow-sm">
-  <p class="h5 my-0 me-md-auto fw-normal">Company name</p>
-  <nav class="my-2 my-md-0 me-md-3">
-    <a class="p-2 text-dark" href="#">Features</a>
-    <a class="p-2 text-dark" href="#">Enterprise</a>
-    <a class="p-2 text-dark" href="#">Support</a>
-    <a class="p-2 text-dark" href="#">Pricing</a>
-  </nav>
-  <a class="btn btn-primary" href="#">Sign up</a>
-</header>
+<svg xmlns="http://www.w3.org/2000/svg" style="display: none;">
```

Comment:
need to check logo

Reply:
will be adressed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/380